src/test/shared/sam/schema/schema.test.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getTestSchemas } from '../../schema/schemaTestUtils'
+import { assertDefinitionProperty, assertProperty, assertRef, assertDefinition } from '../../schema/schemaAssertions'
+import { JSONObject, unmarshal } from '../../schema/schemaTypes'
+
+describe('Sam Schema Regression', function () {
+    let samSchema: JSONObject
+
+    before(async function () {
+        ;({ samSchema } = await getTestSchemas())
+    })
+
+    it('Policy Templates are available', function () {
```

Comment:
Generally Mocha test names are written with action first: `it('has Policy Templates')`.
Which reads as 'it has policy templates'.  You'd make the describe block just 'Sam Schema' or something similar. We aren't super consistent with this of course.